src/app.js

```diff
@@ -77,15 +83,15 @@ export default () => {
                   {done ? (
                     'Amazing'
                   ) : (
-                    <span role="img" aria-label="Yes">
```

Comment:
I'm curios... why using `role="img"` here?

Reply:
YES! 
It's so screen readers don't try to read the Unicode inside of it and only read the aria-label
https://tink.uk/accessible-emoji/

Reply:
Oh that's cool, I didn't know about that part

Reply:
We all learned something today :D